### PR TITLE
Fix Variable test in sections.yml

### PR DESCRIPTION
**Pull request by adam-fowler**

### CODE REVIEW

**Review by jgonggrijp — APPROVED** [review]
I believe this change to be correct.

Question to maintainers: JSON spec hasn't been updated, should this happen before or after merging?

**Comment by Danappelxx** [comment]
Can you run `rake` and commit the changes? Aside from that, lgtm

**Comment by adam-fowler** [comment]
@Danappelxx JSON files are now updated

**Pull request merged**
